assets/js/modules/analytics/datastore/report.test.js

```diff
@@ -31,6 +31,7 @@ import {
 import * as fixtures from './__fixtures__';
 
 describe( 'modules/analytics report', () => {
+	const analyticsRegexp = /^\/google-site-kit\/v1\/modules\/analytics\/data\/report/;
```

Comment:
```suggestion
	const analyticsRegexp = /^\/google-site-kit\/v1\/modules\/analytics\/data\/report/;
```

I know this reduces duplication in the code, but I actually think this is an anti-pattern for tests.

In a component or a file this kind of constant would be handy, allowing us to consolidate code and use this request regex throughout the file without typing it much.

But for a test, it is _much_ more readable to have a test and as much of its data as possible be readable within the test spec's function. Granted, our fixtures require looking at them in a different file (and in modern editors like VSCode getting to this regex is an `Alt` + Click away), but having the URL expected right in the test can help when diagnosing failures and requires less mental energy to inspect/debug/reason about.

The aim of a test spec and test files in general should not be DRY ("don't repeat yourself"). The test should be as illustrative as possible when it fails, and including as much info as close to the test as possible allows for this. Yes, it makes _test refactoring_ more work, but it makes _code refactoring less work_, which is at least part of the benefits of good tests.

So I think this should be removed and these regex's placed back into the test functions themselves. Yes, it's more verbose and repetitive, but it means you can easily, on a screen, get a good sense of what's happening in the test, which URLs we're mocking, etc.